src/App.tsx

```diff
@@ -1,61 +1,38 @@
 import './App.scss';
+import { useState } from 'react';
+import { getPreparedTodos } from './api';
+import { TodoList } from './components/TodoList';
+import { AddForm } from './components/AddForm';
+import { PreparedTodo } from './types/PreparedTodo';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export const initialTodos = getPreparedTodos();
+
+function getNewTodoId(todos: PreparedTodo[]) {
+  const maxId = Math.max(
+    ...todos.map(todo => todo.id),
+  );
+
+  return maxId + 1;
+}
```

Comment:
I think it can be a helper function. You don't need to describe it in components.